### DISCUSSION TITLE
Spectrum OLED not working

### DISCUSSION THREAD

**Discussion by Dave0616** [rern/rAudio#1797] [Fixed Issues]
Im having the same issue that was posted here

https://github.com/rern/rAudio/discussions/763

I have tried everything I can think of, reinstalled, tried all settings, tried another screen and it still wont work right, I can see the logo but as soon as I try to play music the screen goes blank

**Comment by rern** [comment]
Please try: `pacman -Sy cava`

**Reply by Dave0616** [reply]
That seems to have fixed it thank you!
